spec/features/user_can_sumbit_and_view_posts_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.feature 'Post display', type: :feature do
+  scenario 'User sumbits a post' do
+    visit '/posts'
+    click_link 'New post'
+    fill_in 'Message', with: 'Hello, world!'
+    click_button 'Submit'
+    expect(page).to have_content('Hello, world!')
+  end
+
+  scenario 'User submits multiline post' do
+    visit '/posts'
```

Comment:
Heads up this test will break when we add the validation that user must be signed in to access the posts page